### PR TITLE
fixed shortened links with real links

### LINKED ISSUE #199: Replace shortened links with real links
https://github.com/FNNDSC/ChRIS_store_ui/blob/0d150714d6dada40831251796d49e735f6fa756c/src/components/Developers/components/Instructions/Instructions.js#L10

- the link here is outdated
- URL shorteners do not respect privacy

This needs to be changed to https://github.com/FNNDSC/cookiecutter-chrisapp/wiki/Quickstart

### PR DESCRIPTION
**Pull request by NikhilSharma03**
This PR fixes #199

## Description
 
Updated the outdated shortened link in the Instructions.js with real link.

### CODE REVIEW

File: src/components/Developers/components/Instructions/Instructions.js [hunk 1]
```diff
@@ -138,9 +138,9 @@ const Instructions = () => (
             </h3>
             <h3>
               Look at this{' '}
-              <a href="http://bit.ly/2yzbRzF">simple <strong>fs</strong> plugin</a>
+              <a href="https://github.com/FNNDSC/cookiecutter-chrisapp/wiki/Quickstart">simple <strong>fs</strong> plugin</a>
```
Comment (jennydaman) [inline src/components/Developers/components/Instructions/Instructions.js:141]: This is supposed to be a link to https://github.com/FNNDSC/pl-simplefsapp

File: src/components/Developers/components/Instructions/Instructions.js [hunk 2]
```diff
@@ -138,9 +138,9 @@ const Instructions = () => (
...
               {' '}or this{' '}
-              <a href="http://bit.ly/2KbHosS">simple <strong>ds</strong> plugin</a>
+              <a href="https://github.com/FNNDSC/cookiecutter-chrisapp/wiki/Quickstart">simple <strong>ds</strong> plugin</a>
```
Comment (jennydaman) [inline src/components/Developers/components/Instructions/Instructions.js:143]: This is supposed to be a link to https://github.com/FNNDSC/pl-simpledsapp

**Review by jennydaman — CHANGES_REQUESTED** [review]
Not all of the links are the same, would you double-check them?

**Comment by NikhilSharma03** [comment]
@jennydaman .I have updated. Can you check if there's any wrong link?

File: src/components/Developers/components/Instructions/Instructions.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ const Instructions = () => (
               <ul>
                 <li>
                   <a
-                    href="http://bit.ly/2Iih52m"
+                    href="https://github.com/FNNDSC/cookiecutter-chrisapp/wiki/Quickstart"
```
Comment (jennydaman) [inline src/components/Developers/components/Instructions/Instructions.js:36]: Hmm, it's kind of silly for a bulleted list to show two links which go to the same place.
On review, the `<ul>` doesn't seem useful. Perhaps it should be removed altogether.

File: src/components/Developers/components/Instructions/Instructions.js [hunk 4]
```diff
@@ -153,7 +153,7 @@ const Instructions = () => (
           <div className="instructions-body">
             <h3>
               Once you{"'"}ve developed and properly tested your plugin app
-              consult the <a href="http://bit.ly/2ltTJ0w">wiki</a> to learn how to register it to
+              consult the <a href="https://github.com/FNNDSC/cookiecutter-chrisapp/wiki">wiki</a> to learn how to register it to
```
Comment (jennydaman) [inline src/components/Developers/components/Instructions/Instructions.js:156]: Would you link to the specific page, https://github.com/FNNDSC/cookiecutter-chrisapp/wiki/Register-a-new-plugin-app-with-the-ChRIS-store-API

**Review by jennydaman — CHANGES_REQUESTED** [review]

File: src/components/Developers/components/Instructions/Instructions.js (hunk 3 above)
Comment (NikhilSharma03) [reply src/components/Developers/components/Instructions/Instructions.js:36]: @jennydaman Should I deleted <ul> . If that's so then I guess we can remove its parent ( <h3> ) then.

**Pull request closed without merging**